shells/web-shell/elements/pipes/device-client-pipe.js

```diff
@@ -112,10 +112,12 @@ class DeviceClientPipe extends Xen.Debug(Xen.Async, log) {
   update({userid, context, suggestions, arc, pipearc}, state) {
     if (pipearc && !state.pipeStore) {
       state.pipeStore = pipearc._stores[0];
-      log('got pipeStore', state.pipeStore);
-      // retry
       if (!state.pipeStore) {
+        // retry
+        log('looking for pipestore...');
```

Comment:
If there isn't already one, can you file an issue for Runtime team to request a hook we can watch for once stores are loaded?

Reply:
I'll look into it. 

Fwiw, the `web-shell` version of `device-client-pipe` is now deprecated for `pipe-shell`, so this entire module is about to go away.